common/src/main/java/org/apache/auron/conf/AuronBoxedConfiguration.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.auron.conf;
+
+/**
+ * Auron boxed configuration.
+ * It is used to get the config from the real engine configuration, such as spark, flink etc.
+ */
+public interface AuronBoxedConfiguration {
```

Comment:
this interface is likely not used

Reply:
We reuse the config object from the upper-level engine, so we need an abstraction layer within the core module to help retrieve the configuration required by Auron. Each engine should inherit this interface and implement the relevant methods.